docs/content/concepts/ops-jobs-graphs/ops.mdx

```diff
@@ -1,42 +1,41 @@
 ---
 title: Ops | Dagster
-description: Ops are the functional unit of computation in Dagster and contain the core logic of your orchestration graph.
+description: Ops are the atomic unit of computation in Dagster and contain the core logic of your orchestration graph.
```

Comment:
Do we like the word "atomic"? It might imply transactionality that we don't actually have.

Reply:
I have a bit reservations on the work "atomic" - given that many of our users (and prospective users) are familiar with DB systems, it could imply atomic transaction and therefore cause confusion.

Reply:
Good point re "atomic"-- I went with this term to imply "no smaller unit of composition". I changed it bc I didn't really understand what "functional" meant. Some alternatives modifiers for "unit" are "core" or "basic", or nothing at all. IMO these communicate less than "atomic", but also avoid the transactionality baggage.

Reply:
"core" works for me.